format_example.json

```diff
@@ -0,0 +1,135 @@
+{
+  "id": "111111-111111-111111-111111-111111",
+  "name": "Example Project",
+  "summary": "This is a short, one-sentence summary of the Example Project.",
+  "description": "This is a long description describing the project. <b>HTML</b> can optionally be used.",
+  "media": [
+    {
+      "rel": "icon",
+      "uri": "https://example.com/example.png",
+      "sha256": "< sum goes here >"
+    },
+    {
+      "type": "video",
+      "embed": "https://youtube.com/embed...",
+      "caption": "This is an example embed."
+    },
+    {
+      "type": "video",
+      "src": "https://example.com/video.webm",
+      "caption": "This is an example direct video link.",
+      "sha256": "< sum goes here >"
+    },
+    {
+      "type": "image",
+      "src": "https://example.com/image.png",
+      "caption": "This is an example image."
+    }
+  ],
+  "authors": [
+    {
+      "team": true,
+      "id": "111111-111111-111111-111111-111111",
+      "name": "MCIP Contributors",
+      "members": [
+        {
+          "id": "111111-111111-111111-111111-111111",
+          "name": "Example MCIP Member",
+          "uri": "https://example.com/example-webpage"
+        }
+      ]
+    }
+  ],
+  "categories": [
+    {
+      "name": "Example Category",
+      "uri": "https://example.com/browse?category=example",
+      "icon": "https://example.com/example-category.png"
+    }
+  ],
+  "license": {
+    "spdx": "MIT",
```

Comment:
Only common open source licenses have SPDX identifiers, so we will also need to:

* Define an extension to our version of SPDX which lists more licenses, including "unrecognised open licence" and "unrecognised proprietary license" as well as things like MMPL
* Add a name as well as SPDX. I think SPDX should be mandatory, since you can always use an unrecognised license identifier if we define one. I don't think SPDX defines one.

Reply:
Added MMPL as well as Generic-Open and Generic-Proprietary licenses - are there any more that should be added?